transactionoutbox-core/src/test/java/com/gruelbox/transactionoutbox/acceptance/AbstractAcceptanceTest.java

```diff
@@ -406,6 +402,48 @@ final void retryBehaviour() throws Exception {
         });
   }
 
+    @Test
```

Comment:
think my formatting's off from yours - 😬 do you have a formatting config file that can be checked in?

Reply:
Just run `mvn coveo:fmt`.